### PR TITLE
Tests added

### LINKED ISSUE #17: Unit test writing
# REQUIREMENTS
The unit tests have to be written and passed.

# Acceptance Criteria
List all required acceptance criteria
- [ ] Rspec has to be configured
- [ ] Tests have to be written
- [ ] Test have to be running

### PR DESCRIPTION
**Pull request by LPMendoza**
# Tests
The unit tests were written to test differents methods of the controllers.

### CODE REVIEW

**Review by MiguelOcegueraM — APPROVED** [review]
Everything seems to be going very well in my opinion, I like the code very neat!

**Review by glorianayeli — APPROVED** [review]
Good work

File: Gemfile [hunk 1]
```diff
@@ -34,6 +34,7 @@ gem "bootstrap_form", "~> 4.0"
 gem 'devise'
 gem 'activeadmin'
 gem 'arctic_admin'
+gem 'rexml'
```
Comment (LDVera) [inline Gemfile:37]: is interesting the use of this gem in the project :o

**Review by LDVera — APPROVED** [review]

**Review by jnegrete15 — APPROVED** [review]

**Review by n0b0dy-su — APPROVED** [review]
# nice :+1:

**Pull request merged**